benchexec/tools/ceagle.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
I think this is unnecessary because the file is not an executable python script anyway,